Lab 3 AT 9-10-21.Rmd

```diff
@@ -10,10 +13,14 @@ knitr::opts_chunk$set(echo = TRUE)
 Step 1- Read the Data 
 
 ```{r}
+# LOAD THE R PACKAGES!
+library(tidyverse)
+library(leaflet)
```

Comment:
When trying to run this RMarkdown file, I realized you forgot to load these R packages. Make sure the document compiles before submitting it :)!

Reply:
Ok! Thanks